interfaces/Config/templates/config_switches.tmpl

```diff
@@ -269,11 +269,6 @@
                     </div>
                     <!--#end if#-->
                     <div class="field-pair">
-                        <label class="config" for="enable_meta">$T('opt-enable_meta')</label>
```

Comment:
The question is more: why would a user ever turn it off? Using google and the forums, only once in all these years you suggested to a user to turn this off but that didn't even solve the problem.

So this is something that should just happen under the hood, so to say.